Sources/PackageDescription4/LanguageStandardSettings.swift

```diff
@@ -54,13 +53,20 @@ public enum SwiftVersion {
     @available(_PackageDescription, introduced: 5)
     case v5
 
-    /// User-defined value of Swift version.
+    /// A user-defined value for the Swift version.
     ///
-    /// The value is passed as-is to Swift compiler's `-swift-version` flag.
+    /// The value is passed as-is to the Swift compiler's `-swift-version` flag.
     case version(String)
 }
 
 extension SwiftVersion: Encodable {
+
+    /// Encodes this Swift version into the given encoder.
```

Comment:
Should we really provide explicit documentation for Encodable conformance? IIRC it's automatically derived from the documentation in the protocol.

Reply:
You're right. I'll get rid of the docs for Encodable conformance for the language versions as they display just fine, for example at https://developer.apple.com/documentation/swift_packages/clanguagestandard.